pkg/sync/file/filepath_sync.go

```diff
@@ -0,0 +1,138 @@
+package file
+
+import (
+	"context"
+	"encoding/json"
+	"errors"
+	"fmt"
+	"os"
+	"strings"
+
+	"github.com/open-feature/flagd/pkg/sync"
+
+	"gopkg.in/yaml.v3"
+
+	"github.com/fsnotify/fsnotify"
+	"github.com/open-feature/flagd/pkg/logger"
+)
+
+type Sync struct {
+	URI          string
+	Logger       *logger.Logger
+	ProviderArgs sync.ProviderArgs
+	// FileType indicates the file type e.g., json, yaml/yml etc.,
+	fileType string
+}
+
+//nolint:funlen
+func (fs *Sync) Sync(ctx context.Context, dataSync chan<- sync.DataSync) error {
+	fs.Logger.Info("Starting filepath sync notifier")
```

Comment:
Let's use lower case in all log statments.

Reply:
I think this fits into a ticket rather than this PR. Flagd logs usually start with uppercase characters. This is why I used uppercase here.

Reply:
I created a follow up to focus on full soruce #298. Another good first task for a contributor :)